lms/static/certificates/css/style-application.css

```diff
@@ -0,0 +1,1489 @@
+@import url(https://fonts.googleapis.com/css?family=Open+Sans:400,300,600,700);
```

Comment:
@marcotuts and @mattdrayer, should the Sass that produces this be added to the collect static/Django asset pipeline so that this file does not need to be added?

One thing to note is that this is compiled using different (up-to-date) versions of our Bourbon and Neat vendor files. I can downgrade them, but its nice that we are up to date here.